docs/scripts/formattedTSDemos.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Transpiles and formats TS demos.
+ * Can be used to verify that JS and TS demos are equivalent. No introduced change
+ * would indicate equivalence.
+ */
+const childProcess = require('child_process');
```

Comment:
Any reason for not using ES modules?

Reply:
It would require `babel-node` without adding any benefit with the node runtime. Not sure what the state of esm, mjs and node is but I don't see the value in going down that rabbit hole.